code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -856,19 +856,19 @@
 	if(capacitor)
 		phasing_energy_drain = initial(phasing_energy_drain) / capacitor.rating
 		melee_energy_drain = initial(melee_energy_drain) / capacitor.rating
-		light_energy_drain = initial(light_energy_drain) / capacitor.rating
+		light_power_drain = initial(light_power_drain) / capacitor.rating
 	else
 		phasing_energy_drain = initial(phasing_energy_drain)
 		melee_energy_drain = initial(melee_energy_drain)
-		light_energy_drain = initial(light_energy_drain)
+		light_power_drain = initial(light_power_drain)
 
 /// Toggle lights on/off
 /obj/vehicle/sealed/mecha/proc/toggle_lights(forced_state = null, mob/user)
 	if(!(mecha_flags & HAS_LIGHTS))
 		if(user)
 			balloon_alert(user, "mech has no lights!")
 		return
-	if((!(mecha_flags & LIGHTS_ON) && forced_state != FALSE) && get_charge() < light_energy_drain)
+	if((!(mecha_flags & LIGHTS_ON) && forced_state != FALSE) && get_charge() < power_to_energy(light_power_drain, SSobj))
```

Comment:
```suggestion
	if((!(mecha_flags & LIGHTS_ON) && forced_state != FALSE) && get_charge() < power_to_energy(light_power_drain, scheduler = SSobj))
```

Just in case someone in the future decides to change the helper arguments for whatever reason. Wouldn't want another explosive grilles incident or whatever.